engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/api/IGetParameterDefinitionTask.java

```diff
@@ -159,4 +159,20 @@ public interface IGetParameterDefinitionTask extends IEngineTask {
 	 */
 	Collection getSelectionTreeForCascadingGroup(String parameterGroupName);
 
+	/**
+	 * Enable or disable the normal flow mode. The normal flow mode can be used to
+	 * apply filters on data sets in data engine execution.
+	 *
+	 * @param normalFlowModeEnabled Set to true to enable the normal flow mode.
+	 */
+	void setNormalFlowModeEnabled(boolean normalFlowModeEnabled);
```

Comment:
Add "default"-method for the new method - PR #1970, see IWordWriter.

![grafik](https://github.com/user-attachments/assets/641302a4-ea3e-4171-99b3-73e65e3553a5)